dev-guide/featuresets.md

```diff
@@ -94,39 +94,61 @@ The threshold to merge includes referencing the reliability of existing tests.
 This is not a technical limitation — it’s just how our support lifecycle works.
 
 ##  I'd like to declare a feature Accessible-by-default.  What is the process?
-There are a few ways to go about this.
-At a high level they involve proving that the new feature works on all supported architectures, cloud providers,
+
+To make a feature accessible by default, you must prove that the new feature works on all supported architectures, cloud providers,
 network types, and variants (FIPS, RT, cgroupsv2, crun, etc) and has not regressed any other part of OpenShift.
 
 ### Automated regression protection
-Our preferred approach is to use OpenShift CI to prove that your feature is working and that no supported topologies have regressed.
-Writing new e2e tests in openshift/origin is the only automated way to make this claim and ensure that another feature won’t come along and break yours at a later date.
-TechPreview jobs run on AWS, Azure, GCP, and VSphere.
+
+We use OpenShift CI to run tests regularly and prove that features are working and that no supported topologies have regressed.
+Writing new e2e tests either in openshift/origin or within a dedicated periodic test suite, is the only automated way to make this claim and ensure that another feature won’t come along and break yours at a later date.
+
+Dedicated test suites are generally used to demonstrate the reliabililty of long duration or disruptive tests.
+Origin testing is generally preferred and should be the first choice for new tests.
+
+It is expected while in development, that TechPreview testing will provide signal.
+Once promoted, the same signal will come from the stable jobs.
+
+When using a custom periodic suite to provide test data, you must ensure both TechPreview and stable variants exist prior to promotion.
+For origin based tests, TechPreview jobs run on AWS, Azure, GCP, and VSphere.
 
 The textbook example of this process for a complex feature is demonstrated by Egli Hila as he works to ship [workload partitioning](https://github.com/openshift/api/pull/1443).
 In his PR Egli is doing a few powerful things:
 1. He’s created new tests in the E2E suite.
 This is the most powerful automated regression protection we have.
 2. He’s linked directly to the tests in sippy that aggregate results across all the jobs running in tech preview mode (AWS, Azure, GCP, VSphere).
 
-### Obtaining QE sign-off
-Since QE has the ability to fully test features behind feature gates we will allow features to merge that QE has declared done.
+Our automated testing requirements for feature promotion are:
+* At least 5 tests for each feature
+* Tests should run at least 7 times per week
+* Tests should be run on all supported platforms (we inspect AWS, Azure, GCP, vSphere and Bare Metal)
+* Each test must trackable individually with a success or failure metric (typically through JUnit)
```

Comment:
Any more details on how to manage the output for CR consumption or is this assumed to be obvious?

Reply:
Add a link out to https://docs.ci.openshift.org/docs/release-oversight/component-readiness/#how-results-appear-in-component-readiness  below which explains how CR gets the data